### DISCUSSION TITLE
How can I tell what Actions features I can expect in my GHE?

### DISCUSSION THREAD

**Discussion by allwalte** [community/community#40005] [Enterprise]
### Select Topic Area

Question

### Body

I'm setting up some GHA workflows in my local GHE instance, and I keep trying to do things that I find documented, but don't work for me.  And then I look and see, oh, that change was announced 2-3 months ago, so I'm guessing that these are things that have not made it to my local instance yet.  But what I don't know is - are these things *available* to us and we just haven't upgraded yet, or are they just not available yet?  What version are they in?  Etc.  I don't see any clear way of ascertaining this.  (i.e. I want to be able to ask my admin "hey, can we upgrade to version X because it definitely has Y feature that I need")

Is there some way I can tell this?  To give one recent example, I'd like to use a matrix strategy with a reusable workflow, which it looks like was released in late August.

**Comment by ossanna16** [comment]
Hi there @allwalte and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by allwalte** [comment]
I'd still love to know this, never got any nibbles - when reading about a given GHA feature in the docs, how can you know what version of GHE it's in?
